pyiron_atomistics/atomistics/structure/neighbors.py

```diff
@@ -524,6 +526,58 @@ def _check_width(self, width, pbc):
                 return True
         return False
 
+    def get_spherical_harmonics(self, l, m, cutoff_radius=np.inf, rotation=None):
+        """
+        Args:
+            l (int/numpy.array): Degree of the harmonic (int); must have ``l >= 0``.
+            m (int/numpy.array): Order of the harmonic (int); must have ``|m| <= l``.
+            rotation (numpy.array/list): Rotation to make sure phi does not become nan
+
+        Returns:
+            (numpy.array) spherical harmonic values
+
+        Spherical harmonics defined as follows
+
+        Y^m_l(\theta,\phi) = \sqrt{\frac{2l+1}{4\pi} \frac{(l-m)!}{(l+m)!}}
+        e^{i m \theta} P^m_l(\cos(\phi))
+
+        See more on: scipy.special.sph_harm
+
+        """
+        vecs = self.vecs
+        if rotation is not None:
+            vecs = np.einsum('ij,nkj->nki', rotation, vecs)
+        within_cutoff = self.distances<cutoff_radius
+        if np.any(np.all(~within_cutoff, axis=-1)):
+            raise ValueError('cutoff_radius too small - some atoms have no neighbors')
+        phi = np.zeros_like(self.distances)
+        theta = np.zeros_like(self.distances)
+        theta[within_cutoff] = np.arctan2(vecs[within_cutoff,1], vecs[within_cutoff,0])
+        phi[within_cutoff] = np.arctan2(
+            np.linalg.norm(vecs[within_cutoff,:2], axis=-1), vecs[within_cutoff,2]
+        )
+        return np.sum(
+            sph_harm(m, l, theta, phi)*within_cutoff, axis=-1
+        )/np.sum(within_cutoff, axis=-1)
+
+    def get_steinhardt_parameter(self, l, cutoff_radius=np.inf):
+        """
+        Args:
+            l (int/numpy.array): Order of Steinhardt parameter
```

Comment:
Even if we don't expect `cutoff_radius` to be set, it should appear the documented argument list.

Reply:
It's true that it's what reviewing process is for, but I really admire your ability to spot important details XD